lib/ruby-stackoverflow/configuration.rb

```diff
@@ -4,7 +4,7 @@ class Configuration
     attr_accessor :client_id,  :client_sceret , :client_key, :access_token, :api_url
 
     def self.api_url
-      'https://api.stackexchange.com/2.1/'
+      'https://api.stackexchange.com/2.2/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.